generator/integration_tests/golden/v1/internal/golden_kitchen_sink_tracing_connection.cc

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 Google LLC
```

Comment:
Where did these years originate?

Reply:
https://github.com/googleapis/google-cloud-cpp/blob/9196bdcd6281afb62456b648ca436cdfddaf6241/generator/integration_tests/golden_config.textproto#L21

Reply:
Hmm ... we add new output files to the generator and they appear from the past.  Spooky.